### DISCUSSION TITLE
Float type column in Postgres

### DISCUSSION THREAD

**Discussion by LadIQe** [mikro-orm/mikro-orm#1598] [Q&A] [answered]
Hello, want to ask how to define float type column for postgres?

I tried this
```
@Property({ nullable: true, columnType: 'float' })
rating?: number
```

and this

```
@Property({ nullable: true, columnType: 'float8' })
rating?: number
```

even this

```
@Property({ nullable: true, columnType: 'float8', type: 'float' })
rating?: number
```

nothing seems to work. Any help?

Thank you

**Comment by B4nan** [comment, marked as answer]
What do you mean "nothing seems to work"? Because `columnType` is always used as is, and always have priority. I can imagine schema diffing issues, but if you are adding such column, it should work just fine. 

I am in the middle of refactoring of schema diffing, so no plans to change anything on it before I finish with that. But it should greatly help with schema diffing issues. 

Also, in postgres, there is no `float` type - its only an alias for `real` or `double precision`. See https://www.postgresql.org/docs/9.5/datatype-numeric.html#DATATYPE-FLOAT

**Reply by LadIQe** [reply]
I mean, my column is still int4 type. Yeah I know, I tried "real", even this didnt work
